pkg/operation/seed/seed.go

```diff
@@ -808,8 +815,14 @@ func RunReconcileSeedFlow(
 			"networkPolicy": map[string]interface{}{
 				"additionalEgressIPBlocks": additionalEgressIPBlocks,
 			},
+			"eventLogger": map[string]interface{}{
+				"enabled": eventLoggingEnabled,
+			},
+		},
+		"loki": lokiValues,
+		"event-logger": map[string]interface{}{
```

Comment:
I can't seem to find where this value is used. Is it needed?

Reply:
@vlvasilev you resolved the conversation without adding a reply. I think @plkokanov is right here, I cannot see such value under `charts/`. Feel free to correct me if I miss something.

Reply:
sorry I have rather delete the leftovers partially or forgot to save the changes.